data/json_files/release_future_date_change.json

```diff
@@ -0,0 +1,24 @@
+{
+  "uri": "/employmentandlabourmarket/peoplenotinwork/unemployment/datasets/unemploymentbypreviousoccupationunem02",
+  "uri_old": "/an/old/uri",
```

Comment:
`uri_old` is going to do insert and delete so I would remove these from the datastore and they can be added as part of the renaming task.

Reply:
BTW I notice that the release fields are not currently appearing in the JSON when it runs - that's the same in the main branch too.

Reply:
Should I remove the name of the field too or just the value?

Reply:
https://jira.ons.gov.uk/browse/DIS-2064

Reply:
it's not down as `omitempty` so you can either just remove all the values or set it to `omitempty` and remove the key as well.

Reply:
👍 - I've removed all the values and set it to omitempty